functions-methods.js

```diff
@@ -33,4 +69,37 @@
 // checkEmailValidity("tessmellink@novi.nl") geeft true - want @ en punt op de juiste plek
 // checkEmailValidity("n.eekenanovi.nl") geeft false - want geen @
 // checkEmailValidity("n.eeken@novinl.") geeft false - want de punt mag niet als laatst
-// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
\ No newline at end of file
+// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
+
+
+function checkEmailValidity(emailAddress){
+    let emailValidity;
+    let atPosition = emailAddress.lastIndexOf("@");
+    let puntPostion = emailAddress.lastIndexOf(".");
+
+    if (emailAddress.substring(atPosition).includes("@")){
+        if (emailAddress.substring(atPosition).includes(".")){
+            if (emailAddress.substring(puntPostion).length > 1){
+                emailValidity = true;
+            } else { emailValidity = false; }
+        } else { emailValidity = false; }
+    } else { emailValidity = false; }
+
+    return [emailValidity] ;
+}
```

Comment:
Dit had wel een stuk korter gekund. Het maken van de variabelen is top. Je kan echter beter gebruik maak van de && operator i.p.v. een if/else statement.